src/main/java/ua/artcode/udiary/config/ObjectHolder.java

```diff
@@ -0,0 +1,27 @@
+package ua.artcode.udiary.config;
+
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import ua.artcode.udiary.model.User;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ObjectHolder {
+
+    private static Map<String,Object> staticObjects = new HashMap<>();
+
+    static  {
+        staticObjects.put("gson", new GsonBuilder().setPrettyPrinting().create());
+        staticObjects.put("userListType",new TypeToken<List<User>>(){}.getType());
```

Comment:
perhaps we can avoid userListType in map

Reply:
Unfortunately, I dont understatnd the point of this comment. More explanation is needed.